Adapter/Client.php

```diff
@@ -83,7 +85,14 @@ public function addData($indexName, $data)
 
                 $price['default'] = number_format($price['default'], 2);
             }
+
+            foreach ($facets as $facet) {
+                if (isset($item[$facet]) && !is_array($item[$facet])) {
+                    $item[$facet] = [strval($item[$facet])];
```

Comment:
composite products send their attributes in an array - I thought it's probably easier just to force this for everything.